test/test_table.py

```diff
@@ -0,0 +1,147 @@
+from hpack.table import HeaderTable
+import pytest
+import sys
+_ver = sys.version_info
+is_py2 = _ver[0] == 2
+is_py3 = _ver[0] == 3
+
+class TestHeaderTable(object):
+    def test_getitem_dynamic_table(self):
+        tbl = HeaderTable()
+        off = len(HeaderTable.STATIC_TABLE)
+        val = (b'TestName', b'TestValue')
+        tbl.add(val[0], val[1])
+        res = tbl[off + 1]
+        assert res == val
+
+    def test_getitem_static_table(self):
+        tbl = HeaderTable()
+        exp = HeaderTable.STATIC_TABLE[0]
+        res = tbl[1]
+        assert res == exp
```

Comment:
We know what element this should be, let's assert that it's there, rather than checking the way the implementation works.